sparse_strips/vello_hybrid/src/render.rs

```diff
@@ -113,9 +119,7 @@ impl Renderer {
     /// Creates a new renderer
     ///
     /// The target parameter determines if we render to a window or headless
-    pub fn new(device: &Device, _options: &RendererOptions) -> Self {
-        let format = wgpu::TextureFormat::Bgra8Unorm;
```

Comment:
WebGL2 doesn't support `Bgra8Unorm` texture formats. I think it's better to allow consumers to decide target format